ddtrace/appsec/iast/processor.py

```diff
@@ -46,4 +49,8 @@ def on_span_finish(self, span):
         if data:
             span.set_tag_str(IAST_JSON, json.dumps(attr.asdict(data)))
 
+            span.set_tag(MANUAL_KEEP_KEY)
```

Comment:
this will keep the entire trace, is that expected? or do you just want to keep the individual span?

Reply:
Yes it is, we wanna keep the entire trace :+1:

Reply:
Great, thanks for confirming, I just wanted to be sure since it was listed as "keep span if..."